src/AasxServerStandardBib/AdminShellPackageEnv.cs

```diff
@@ -307,7 +307,9 @@ public bool Load(string fn, bool indirectLoadSave = false)
                                         JsonSerializer serializer = new JsonSerializer();
                                         serializer.Converters.Add(new AdminShellConverters.JsonAasxConverter("modelType", "name"));
                                         this.aasenv = (AdminShell.AdministrationShellEnv)serializer.Deserialize(file, typeof(AdminShell.AdministrationShellEnv));
+                                        file.Close();
```

Comment:
Would it be worth considering "using" statements here?

Reply:
Actually, it's a bug without `using` in case of exceptions.

Reply:
Thanks for your comments. You mean "file.Dispose()" instead of "file.Close()"?

Reply:
@magnus-redeker actually, as you use `using`, you don't need to close -- [this line](https://github.com/admin-shell-io/aasx-server/pull/39/files/5e85206076be7d9dc059b140f3a70436aac143fe#diff-cec56812a26e5f682b4e6246533b33a16e21728b0a1640a0ccf9685e6217cbdbL305) and [this line](https://github.com/admin-shell-io/aasx-server/pull/39/files/5e85206076be7d9dc059b140f3a70436aac143fe#diff-cec56812a26e5f682b4e6246533b33a16e21728b0a1640a0ccf9685e6217cbdbR303) are enough. The method `Dispose()` will be automatically called even in case of exceptions.